grid/recruitment/candidates/profile.supr

```diff
@@ -0,0 +1,358 @@
+name = "recruitment/candidates"
+version = "1.0.0"
+
+"""
+Create Candidate
+Creates an application for specific job. The candidate may (depends on the provider implementation) receive notification that her/his application was successfully accepted by the company.
+"""
+usecase CreateCandidate {
+
+  input {
+    """
+    Job Id
+    Id of a job, to which candidate applies to.
+    """
+    jobId! string!
+
+    """
+    First Name
+    First name of a candidate.
+    """
+    firstName! string!
+
+    """
+    Last Name
+    Last name of a candidate.
+    """
+    lastName! string!
+
+    """
+    Email
+    Email of a candidate.
+    """
+    email! string!
+
+    """
+    Phone
+    Phone number of a candidate.
+    """
+    phone string
+
+    """
+    Address
+    Address of a candidate.
+    """
+    address string	
+
+    """
+    Education
+    Array of education experiences.
+    """
+    education [Education]
+
+    """
+    Work Experience
+    Array of work experiences.
+    """
+    workExperience [WorkExperience]
+
+    """
+    CV (Resume)
+    Object containing either url to a CV or data of a CV.
+    """
+    cv { url string } | { name string, data string }
+
+    """
+    Links
+    Array of links to different profiles or portfolios of a candidate.
+    """
+    links [Link]
+
+    """
+    Answers
+    Array of answers to questions connected to a job that candidate applies to.
+    """
+    answers [Answer]
+
+    """
+    Stage Id
+    Unique identifier of a stage, in which candidate is in.
+    """
+    stageId string
+  }
+
+  result {
+    """
+    Id
+    Unique identifier of newly created candidate.
+    """
+    id! string!
+
+    """
+    Job Id
+    Unique identifier of a job, newly created candidate applied for.
+    """
+    jobId! string!
+  }!
+
+  error Error
+}
+
+"""
+Education
+Object representing educational experience of a candidate.
+"""
+model Education {
```

Comment:
We should have documented all the fields including Education, WorkExperience and Answer models.